lib/plugins/config/settings/config.metadata.php

```diff
@@ -118,6 +118,7 @@
 $meta['useheading']  = array('multichoice','_choices' => array(0,'navigation','content',1));
 $meta['sneaky_index'] = array('onoff');
 $meta['hidepages']   = array('regex');
+$meta["ack_page_nearest"] = array('onoff');
```

Comment:
```suggestion
$meta['ack_page_nearest'] = array('onoff');
```

Reply:
Updated in latest commit to match the style.